tests/unit/networking.tcl

```diff
@@ -37,8 +37,40 @@ test {CONFIG SET bind address} {
     }
 } {} {external:skip}
 
+# Attempt to connect to host using a client bound to bindaddr,
+# and return a non-zero value if successful within specified
+# millisecond timeout, or zero otherwise.
+proc test_loopback {host bindaddr timeout} {
+    if {[exec uname] != {Linux}} {
```

Comment:
Is there any real reason to skip this on non Linux platforms? now that we have that test and we think it's reliable, maybe it's worthwhile to test other platform, maybe we'll find bugs in Redis?

Reply:
I considered this option, but thought it's still a potential for more noise. The real purpose of this test was to make sure we're handling the configuration properly, that can be achieved on a single platform.